packages/client/src/components/Answers.tsx

```diff
@@ -37,29 +37,56 @@ const Answers: FunctionComponent<AnswersProps> = ({
 
   return (
     <ComponentWrapper data-testid={QUESTION_ANSWERS}>
-      <RadioGroup name={id}>
-        {answers?.map((answer, index) => {
-          const { label, formValue, value } = answer;
+      {question.collection ? (
```

Comment:
I've also made quite some changes to this file, can you please help me to avoid conflicts by changing this code a bit? Please first define these variables and render the Components conditionally based on the vars instead of your ternary:

```
const showCheckboxInput = collection;
const showRadioInput = !showCheckboxInput;
```

```
{showCheckboxInput && ....}
{showRadioInput && <RadioGroup name={id}>....</RadioGroup>}
```

My PR will also support `showTextInput` and in the future we can extend this with other inputs. I will also work on converting the checkboxes to support ASC-UI, in another PR.